### PR TITLE
some pylint fixes

### PR DESCRIPTION
**Pull request by wdpypere**
mainly implement f-strings. Not finished by a long shot. As an unexpected side effect, about 5 % faster on my machine.

### CODE REVIEW

File: lib/vsc/install/commontest.py [hunk 1]
```diff
@@ -36,10 +36,10 @@
 
 import optparse
 import os
-import pkg_resources
 import pprint
 import re
 import sys
+import pkg_resources
```
Comment (itkovian) [inline lib/vsc/install/commontest.py:42]: Any reason not to keep this alphabetical?

Comment (wdpypere) [reply lib/vsc/install/commontest.py:42]: pylint says the order is wrong

Comment (itkovian) [reply lib/vsc/install/commontest.py:42]: why?

Comment (wdpypere) [reply lib/vsc/install/commontest.py:42]: dependencies between the modules I assume.

Comment (itkovian) [reply lib/vsc/install/commontest.py:42]: fair enough.

**Pull request merged**